pmm-app/tests/verifyPMMInventory_test.js

```diff
@@ -0,0 +1,179 @@
+Feature('Inventory page');
+
+Before(async (I) => {
+  I.Authorize();
+});
+
+Scenario(
+  'PMM-T371 - Verify sorting in Inventory page(Services tab) @not-pr-pipeline @not-ui-pipeline @nightly',
+  async (I, pmmInventoryPage) => {
+    I.amOnPage(pmmInventoryPage.url);
+    I.waitForVisible(pmmInventoryPage.fields.tableRow, 20);
+    const countOfRows = await I.grabNumberOfVisibleElements(pmmInventoryPage.fields.tableRow);
+    let serviceNames = new Array(countOfRows);
+    let forSort = new Array(countOfRows);
```

Comment:
Those two variables should still be `const`. Whenever you are manipulations the array elements without the intent to reassign the array itself => it's a const. Sort of a guard to inadvertent reassignment.